stack.yaml

```diff
@@ -46,7 +46,7 @@ packages:
   extra-dep: true
 - location:
     git: https://github.com/serokell/time-warp-nt.git
-    commit: dcc41ea61e32e74f005cdd40ea774c210e6cfe45
+    commit: 7e1c8d794269884fa8740a347eece701a4efa1e8
```

Comment:
No, the policy is that all dependencies should be in protected branches (master or release branch).